dbt_rpc/rpc/task_manager.py

```diff
@@ -87,7 +89,12 @@ def __init__(self, args, config, task_types: TaskTypes) -> None:
         self.manifest: Optional[Manifest] = None
         self._task_types: TaskTypes = task_types
         self.active_tasks: TaskHandlerMap = {}
-        self.gc = GarbageCollector(active_tasks=self.active_tasks)
+        gc_settings = GCSettings(
+            maxsize=GC_MAXSIZE,
+            reapsize=GC_REAPSIZE,
+            auto_reap_age=timedelta(hours=GC_AUTO_REAP_HOURS)
+        )
```

Comment:
Do you mean `GarbageCollector.__init__`? I took it out of there since there's that if settings condition that's never used. I don't have a preference of putting it here vs in the `GarbageCollector.__init__` and letting it get initialized there, but feels like this is more straightforward to read.

Reply:
Makes sense to me!